### DISCUSSION TITLE
Picture File Locations in Dashboard cards: Use the Settings: Areas & Zones drag/drop dialog.

### DISCUSSION THREAD

**Discussion by philpownall** [home-assistant/frontend#18669] [Dashboard]
There are a large number of posts related to picture file locations for the various picture element dashboard cards - the www directory, setting it up, file paths, and so on.  But I see you have already solved this problem for Settings -> Areas & Zones -> +Create Area, which has a lovely pop-up drag&drop picture section.  This can be used to add pictures from cell phones using the companion app, or from any computer, without having to cut/paste/modify file locations.

So, why not migrate this pop-up from the Create  Area card to all of the Picture cards in the Lovelace dashboards?  I see that this pop-up copies the pic to a new directory under /root/config/image which is accessed by the Area card, thereby solving the problem handily with no manual input of file locations.

**Comment by karwosts** [comment]
Sounds like a great idea to me. #18695 as a starting point.
